monitor/host_monitoring.go

```diff
@@ -25,7 +26,11 @@ type hostMonitoringFunc func(*evergreen.Settings) []error
 // monitorReachability is a hostMonitoringFunc responsible for seeing if
 // hosts are reachable or not. returns a slice of any errors that occur
 func monitorReachability(settings *evergreen.Settings) []error {
-	grip.Info("Running reachability checks...")
+	grip.Info(message.Fields{
+		"runner":   RunnerName,
+		"function": "monitorReachability",
```

Comment:
We use "operation" elsewhere. We should probably pick one.

Reply:
changed